fe/fe-core/src/main/java/com/starrocks/analysis/ShowWorkGroupStmt.java

```diff
@@ -0,0 +1,21 @@
+package com.starrocks.analysis;
+
+import com.starrocks.catalog.WorkGroup;
+import com.starrocks.qe.ShowResultSetMetaData;
+
+public class ShowWorkGroupStmt extends ShowStmt {
```

Comment:
Why need show work group and show work groups at the same time?

Reply:
ShowWorkGroupsStmt handles two stmt 
1. `show resource_groups all`,   list all workgroups;
2. `show resource_groups`, only list all workgroups visible to current user.

ShowWorkGroupStmt handles 
1. `show resource_group rg0`, just show workgroup named rg0

Reply:
stmt  could support where to filter.

Keep `ShowWorkGroupsStmt` and `ShowWorkGroupStmt` at  the same time is seems strange.

Reply:
use only one ShowWorkGroupStmt?

Reply:
ShowWorkGroupsStmt.  ShowWorkGroupStmt could use ShowWorkGroupsStmt + where

Reply:
ShowWorkGroupsStmt is removed, ShowWorkGroupStmt can handle all cases of show